Sprint-1/refactor/includes.js

```diff
@@ -1,13 +1,5 @@
-// Refactor the implementation of includes to use a for...of loop
-
-function includes(list, target) {
-  for (let index = 0; index < list.length; index++) {
-    const element = list[index];
-    if (element === target) {
-      return true;
-    }
-  }
-  return false;
+function includes(arr, target) {
+  return arr.includes(target);
```

Comment:
While this approach works, it does not use a construct specified in the spec (line 1).